IncBackups/IncBackupsControl.py

```diff
@@ -57,9 +58,9 @@ def run(self):
             self.restorePoint()
 
     def getRemoteBackups(self):
-        if self.backupDestinations[:4] == 'sftp':
-            path = '/home/backup/%s' % (self.website)
-            command = 'export RESTIC_PASSWORD=%s PATH=${PATH}:/usr/bin && restic -r %s:%s snapshots' % (
+        if self.backupDestinations == 'onedrive':
```

Comment:
why did you change this to onedrive and not rclone?

Reply:
I explained this in a variety of comments here and in issue #834. 

This line refers not to rclone, but to the name of the rclone remote that you want to use. In this case, I was only setting it up to work with onedrive. Ideally we will change it so that it reads `self.backupDestinations = selectedRcloneRemote`. And that variable will contain the name of the rclone remote that has been added as a backup destination and was selected in the create backup window. But we don't have that mechanism yet, so I just hardcoded it as onedrive. 

For now, you can call it anything - onedrive, rclone, etc... The important part is that you can test this whole mechanism with ANY rclone provider, IF you name the remote "onedrive" (or whatever the string is in this line. 

For example, if you change it to "rclone", you would have to create a rclone remote with your desired cloud provider (google drive, onedrive, dropbox, etc) named rclone.

Does this make sense? It's just a placeholder to make it work until a mechanism is made for the front end to add multiple rclone remotes (that were created in ssh) as destinations for Inc backups, and to select them from the create backup page.

Also, once again, I've overridden the entire SFTP mechanism, which I assume you don't want to do. It's just for testing purposes and you can use it as an example to create a separate rclone mechanism alongside local, SFTP and aws. Or, if you think it is better, you can replace some or all of those with rclone.